vocabularies/Common.json

```diff
@@ -1288,6 +1288,51 @@
       "@Common.Experimental": true,
       "@Core.IsURL": true,
       "@Core.Description": "Base URL for WebSocket connections"
+    },
+    "RecommendationsAction": {
+      "$Kind": "Term",
+      "$AppliesTo": ["EntityType"],
+      "@Common.Experimental": true,
+      "@Core.Description": "Name of an action (or function) import that retrieves AI-based recommendations (see [template](#Template_RecommendationsAction)) for an entity of the annotated type"
+    },
+    "Template_RecommendationsAction": [
+      {
+        "$Kind": "Action",
+        "@Common.Experimental": true,
+        "@Core.Description": "Template for actions that retrieve AI-based recommendations for an entity and its related entities",
+        "@Core.LongDescription": "The template action itself cannot be invoked.",
+        "$Parameter": [
+          {
+            "$Name": "EntityKeys",
```

Comment:
So the action will return recommendations for all properties of the entities for which recommendations are calculated. But the underlying assumption of not providing a mean to limit the properties is that the coupling between service and UI is tightly enough so that there won't be (many) recommendations generated for unavailable fields?